integration-tests/opentelemetry-jaeger-remote/src/test/java/io/quarkus/it/opentracing/OTelJaegerRemoteTest.java

```diff
@@ -0,0 +1,131 @@
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.transport.DockerHttpClient;
+import com.github.dockerjava.zerodep.ZerodepDockerHttpClient;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Scope;
+import io.quarkus.test.junit.QuarkusTest;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+@QuarkusTest
+public class OTelJaegerRemoteTest {
+
+    private static final OkHttpClient client = new OkHttpClient();
+    private static final int QUERY_PORT = 16686;
+    private static final int COLLECTOR_PORT = 14250;
+    private static final int HEALTH_PORT = 14269;
+    private static final String JAEGER_URL = "http://localhost";
+    @Inject
+    OpenTelemetry openTelemetry;
+    private static final DockerClient dockerClient;
+
+    static {
+        DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder().build();
+        DockerHttpClient httpClient = new ZerodepDockerHttpClient.Builder()
+                .dockerHost(config.getDockerHost())
+                .sslConfig(config.getSSLConfig())
+                .build();
+        dockerClient = DockerClientImpl.getInstance(config, httpClient);
+        if (dockerClient.listContainersCmd().exec().stream()
+                .noneMatch(container -> container.getNames()[0].equals("/jaeger"))) {
+            ExposedPort queryPort = ExposedPort.tcp(QUERY_PORT);
+            ExposedPort collectorPort = ExposedPort.tcp(COLLECTOR_PORT);
+            ExposedPort hostPort = ExposedPort.tcp(HEALTH_PORT);
+            Ports portBindings = new Ports();
+            portBindings.bind(queryPort, Ports.Binding.bindPort(QUERY_PORT));
+            portBindings.bind(collectorPort, Ports.Binding.bindPort(COLLECTOR_PORT));
+            portBindings.bind(hostPort, Ports.Binding.bindPort(HEALTH_PORT));
+            CreateContainerResponse container = dockerClient
+                    .createContainerCmd("ghcr.io/open-telemetry/opentelemetry-java/jaeger:1.32")
+                    .withExposedPorts(queryPort, collectorPort, hostPort)
+                    .withHostConfig(newHostConfig().withPortBindings(portBindings))
+                    .withName("jaeger")
+                    .exec();
+            dockerClient.startContainerCmd(container.getId()).exec();
+        }
+    }
+
+    @AfterAll
+    static void teardown() {
+        dockerClient.listContainersCmd().exec()
+                .forEach(container -> {
+                    dockerClient.stopContainerCmd(container.getId()).exec();
+                    dockerClient.removeContainerCmd(container.getId()).exec();
+                });
+    }
+
+    @Test
+    void testJaegerRemoteIntegration() {
+        createTestSpan(openTelemetry);
+        Awaitility.await()
```

Comment:
This test will be flaky. You need to fire a few traces and count the ones arriving must be equal or above the expected sample. 
You have a sample rate of 0,9, meaning, this test, like it is now will fail  1 each 10 runs

Reply:
On the sampling rate, should I keep it to 1.0?

Reply:
You can but you should find a way to check that this sampler is actually being used.

Reply:
One more thing that I didn't understand from your comment is if I fire a few traces then how will I get arriving above the expected sample? Wouldn't it be equal or less?

Reply:
I was thinking that if you fire 10 spans, with 0.9 sampling, only 9 should arrive... But this is not deterministic and therefore, also flaky...
I'd say, fire 1 span and make the sample arg be 1.0. Also find a way to know which sampler is being used... Most likely a different test, on the deployment tests, will be easier.

Reply:
I got it now. Will try to incorporate changes in these tests only but if doesn't work then will create a separate one.

Reply:
Have updated the tests